package.json

```diff
@@ -44,7 +45,7 @@
     "download": "^8.0.0",
     "fs-extra": "^10.0.0",
     "multer": "^1.4.2",
-    "path-to-regexp": "^6.2.0",
+    "path-to-regexp": "^0.1.7",
```

Comment:
This is so it's the same version installed by everything else. I was getting issues where the wrong version was being picked up, and the exports are different between the versions so it would cause builds to break